include/ardpolyfills/BoardData.cxx

```diff
@@ -0,0 +1,52 @@
+/*
+ *  BoardData.cxx
```

Comment:
Wrong file; this should be part of some `Arduino.cxx` in module ardrivo (check branch ard-cpp-trigci to locate this module)

Reply:
I will fix the problems that you mentioned.